vllm/model_executor/models/gemma.py

```diff
@@ -361,8 +361,7 @@ class GemmaForCausalLM(nn.Module, SupportsLoRA, SupportsPP):
         ".v_proj.",
         ".o_proj.",
     ]
-    # in TP, these weights are partitioned along the column dimension (dim=-1)
-    column_parallel_weights_modules = [".down_proj.", ".o_proj."]
+
```

Comment:
nit: remove these newlines to the following bnb related map since this block starts with `# BitandBytes specific attributes`